File: Selenium/Activities/Activity20.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity20 {
    public static void main(String[] args) throws InterruptedException {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/javascript-alerts");
        System.out.println("Title of webpage:" + driver.getTitle());
        driver.findElement(By.id("prompt")).click();
        System.out.println("Text from alert box:"+driver.switchTo().alert().getText());
        driver.switchTo().alert().sendKeys("Awesome!");
        driver.switchTo().alert().accept();
        driver.close();
    }
}

File: Selenium/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support.wait import WebDriverWait
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    wait = WebDriverWait(driver, 10)
    print("Title of webpage:" + driver.title)
    driver.find_element(By.ID, "prompt").click()
    print("Text from alert box:" + driver.switch_to.alert.text)
    driver.switch_to.alert.send_keys("Awesome!")
    driver.switch_to.alert.accept()
    driver.close()
